site/content/includes/m3query/annotated_config.yaml

```diff
@@ -0,0 +1,246 @@
+# The server listen address
```

Comment:
Generally, I think this config needs to be updated. I'm not sure it ever got changed after 1.0. Maybe not something we need to do for this PR, but worth us doing

Reply:
@nbroyles I have that in progress right now, and boy is it a lot of work!